packages/wallet/api/src/lib-wallet.js

```diff
@@ -131,6 +131,22 @@ export function makeWallet({
   const installationMapping = makeMapping('installation');
 
   const brandTable = makeIssuerTable();
+
+  const initBrandToBoardId = async brand => {
+    const { valToBoardId: brandToBoardId } = brandMapping;
+    if (brandToBoardId.has(brand)) {
+      return brandToBoardId.get(brand);
+    }
+
+    const brandBoardId = await E(board).getId(brand);
```

Comment:
why could this call have the side-effect of changing `brandToBoardId.has(brand)` from false to true?

Reply:
It wouldn't, but `brandToBoardId` might be called again with the same brand before this resolves. Tbh I was just following the pattern in `initIssuerToBoardId`.